packages/pwa-kit-react-sdk/src/ssr/universal/utils.ts

```diff
@@ -53,23 +56,31 @@ export const getAssetUrl = (path: string) => {
  * @function
  * @returns {string} The full URL to the static asset.
  */
-export const getStaticAssetUrl = (path: string, opts: GetAssetUrlOptions) => {
+export const getStaticAssetUrl = (path = '', opts: GetAssetUrlOptions) => {
     const {appExtensionPackageName = ''} = opts || {}
+    const extensionPrefix = `${
+        appExtensionPackageName ? `/${EXTENSIONS_NAMESPACE}/${appExtensionPackageName}` : ''
+    }`
 
     /* istanbul ignore next */
-    const publicPath = onClient
+    let publicPath = onClient
         ? // @ts-ignore
           `${window.Progressive.buildOrigin as string}`
-        : `${bundleBasePath}/${process.env.BUNDLE_ID || 'development'}/`
+        : `${bundleBasePath}/${process.env.BUNDLE_ID || 'development'}`
+
+    // Normalize the public path by removing the trailing slash
+    publicPath = publicPath.replace(/\/$/, '')
```

Comment:
nit: Should we extract this logic into a small utility and re-use it instead of duplicating the same logic? 

https://github.com/SalesforceCommerceCloud/pwa-kit/blob/01617982e298b3b75386cd8c4a62b833bc7905e1/packages/pwa-kit-react-sdk/src/ssr/universal/utils.ts#L43